apps/meteor/app/livechat/server/lib/DepartmentsHelper.ts

```diff
@@ -0,0 +1,49 @@
+import { LivechatDepartment, LivechatDepartmentAgents, LivechatRooms } from '@rocket.chat/models';
```

Comment:
Since this is already under /lib, do w need to call it "Helper"? 🤔

Reply:
Lol no, lets change that!